tests/test_modeling_common.py

```diff
@@ -551,10 +551,6 @@ def check_training_gradient_checkpointing(self, gradient_checkpointing_kwargs=No
             return
 
         for model_class in self.all_model_classes:
-            config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
```

Comment:
This was redundant with L565 (from the unmodified file). The lines below, such as `config.use_cache = False`, had no effect since `config` was overwritten in L565